bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -65,6 +72,11 @@ def destroy(self):
             logging.info("Local test cluster is not started")
             return
         self.terminate_process()
+        log_files = walk(os.path.join(self.work_dir, self.install_dir, "logs"))
+        test_recorder_builder = TestRecorderBuilder(self.component_name, self.component_test_config, 0,
```

Comment:
How do we know the exit code is 0, can this be pulled in from somewhere?

Reply:
It is not used anywhere and is only called after the process is terminated. The only reason of adding it was to satisfy the param requirements of TestRecorderBuilder

Reply:
[Discussed Offline] We should just remove the exit code if it isn't being explicitly used